service/controller/aws/v19/version_bundle.go

```diff
@@ -7,26 +7,6 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
-			{
-				Component:   "cert-exporter",
-				Description: "Add toleration for all taints",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "net-exporter",
-				Description: "Add toleration for all taints",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "node-exporter",
-				Description: "Add toleration for all taints",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "nginx-ingress-controller",
-				Description: "Update to 0.25.1. https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/master/CHANGELOG.md",
-				Kind:        versionbundle.KindChanged,
-			},
```

Comment:
These changes shipped in v18 so I think they should be removed.

@tomahawk28 Can you confirm?